src/Bridges/Constraint/bridges/split_equalto.jl

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    SplitEqualToBridge{T,F,G} <: Bridges.Constraint.AbstractBridge
```

Comment:
maybe `SplitComplexEqualToBridge`

Reply:
Agreed

Reply:
Renaming the Zeros one: https://github.com/jump-dev/MathOptInterface.jl/pull/1950